frontend/components/charts/SignedExtrinsicsBarChart.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div>
+    <div v-if="loading" class="text-center py-4">
+      <Loading />
+    </div>
+    <div v-else>
+      <ChartFilter :buttons="filterButtons" :active-button="activeButton" />
+      <BarChart
+        class="canvas-chart"
+        :chart-data="chartData"
+        :options="chartOptions"
+        :height="200"
+      />
+    </div>
+  </div>
+</template>
+<script>
+import BarChart from '@/components/charts/BarChart.js'
+import Loading from '@/components/Loading.vue'
+import ChartFilter from '@/components/ChartFilter.vue'
+import chartsMixin from '@/mixins/chartsMixin.js'
+
+export default {
+  components: {
+    BarChart,
+    Loading,
+    ChartFilter,
+  },
+  mixins: [chartsMixin],
+  data() {
+    return {
+      filterButtons: this.getDefaultFilterButtons(),
+      loading: true,
+      activeButton: '30D',
+      chartData: {},
+      chartOptions: this.getChartOptions(
+        'Cere Signed Extrinsics Chart ',
+        'Extrinsics per Period'
+      ),
+    }
+  },
+  mounted() {
+    this.month()
+  },
+  methods: {
+    async month() {
+      this.activeButton = '30D'
+      const queryLimit = 30
```

Comment:
Move this value to constant and reuse. The same for all buttons.

Reply:
Done